ewatercycle/models/wflow.py

```diff
@@ -0,0 +1,157 @@
+import shutil
+import time
+from os import PathLike
+from pathlib import Path
+from typing import Any, Iterable, Optional, Tuple
+
+import numpy as np
+import xarray as xr
+from cftime import num2date
+from grpc4bmi.bmi_client_docker import BmiClientDocker
+from grpc4bmi.bmi_client_singularity import BmiClientSingularity
+
+from ewatercycle.models.abstract import AbstractModel
+from ewatercycle.parametersetdb.config import CaseConfigParser
+
+# from ewatercycle import CFG
+
+# mock CFG until the CFG PR is merged.
+CFG = {
+    "container_engine": "docker",
+    "singularity_images.wflow": "ewatercycle-wflow-grpc4bmi.sif",
+    "docker_images.wflow": "ewatercycle/wflow-grpc4bmi:latest",
+    "scratch_dir": "./"
+}
```

Comment:
#45 has been merged. Can this be removed and master branch be merged into this branch?

Reply:
Yes, done